diff --git a/dist/esm/index.mjs b/dist/esm/index.mjs
index a27063f68d7619bce4ef39d5819f6c2ebb08169f..730f6607cb727a1b7590fc4abc8eba73c884bcad 100644
--- a/dist/esm/index.mjs
+++ b/dist/esm/index.mjs
@@ -17902,7 +17902,7 @@ var prepareServer = async ({
       }
       return close();
     },
-    serveUrl: `http://localhost:${serverPort}`,
+    serveUrl: `http://127.0.0.1:${serverPort}`,
     offthreadPort: serverPort,
     compositor,
     sourceMap: () => localSourceMap,
diff --git a/dist/prepare-server.js b/dist/prepare-server.js
index 81f412590a414ab838819d1ef864f77b72dfe37f..1f0fe95c3358218bfda02e75f4a3611bcdacf501 100644
--- a/dist/prepare-server.js
+++ b/dist/prepare-server.js
@@ -91,7 +91,7 @@ const prepareServer = async ({ webpackConfigOrServeUrl, port, remotionRoot, offt
         },
         // This should be kept localhost, even if the server is bound to ::1,
         // to prevent "Failed to load resource: net::ERR_FAILED  Access to image at 'http://localhost:3000/proxy?src=http%3A%2F%2F%5B%3A%3A%5D%3A3000%2Fpublic%2Fframer.webm&time=0&transparent=false' from origin 'http://[::]:3000' has been blocked by CORS policy: The request client is not a secure context and the resource is in more-private address space `local`".
-        serveUrl: `http://localhost:${serverPort}`,
+        serveUrl: `http://127.0.0.1:${serverPort}`,
         offthreadPort: serverPort,
         compositor,
         sourceMap: () => localSourceMap,
