diff --git a/dist/cjs/video/offthread-video-source.js b/dist/cjs/video/offthread-video-source.js
index b2616a34744f3994c35632f17c361107ee318c71..04480781366125adb8d3eaf3e9b6a503d8eca45c 100644
--- a/dist/cjs/video/offthread-video-source.js
+++ b/dist/cjs/video/offthread-video-source.js
@@ -3,6 +3,6 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.getOffthreadVideoSource = void 0;
 const absolute_src_1 = require("../absolute-src");
 const getOffthreadVideoSource = ({ src, transparent, currentTime, toneMapped, }) => {
-    return `http://localhost:${window.remotion_proxyPort}/proxy?src=${encodeURIComponent((0, absolute_src_1.getAbsoluteSrc)(src))}&time=${encodeURIComponent(Math.max(0, currentTime))}&transparent=${String(transparent)}&toneMapped=${String(toneMapped)}`;
+    return `http://127.0.0.1:${window.remotion_proxyPort}/proxy?src=${encodeURIComponent((0, absolute_src_1.getAbsoluteSrc)(src))}&time=${encodeURIComponent(Math.max(0, currentTime))}&transparent=${String(transparent)}&toneMapped=${String(toneMapped)}`;
 };
 exports.getOffthreadVideoSource = getOffthreadVideoSource;
diff --git a/dist/esm/index.mjs b/dist/esm/index.mjs
index 7a137152c631b913d8e5aba873d3860450c400d2..32865399da2d67555fc428afc257fe34564e1f2c 100644
--- a/dist/esm/index.mjs
+++ b/dist/esm/index.mjs
@@ -14286,7 +14286,7 @@ var getOffthreadVideoSource = ({
   currentTime,
   toneMapped
 }) => {
-  return `http://localhost:${window.remotion_proxyPort}/proxy?src=${encodeURIComponent(getAbsoluteSrc(src))}&time=${encodeURIComponent(Math.max(0, currentTime))}&transparent=${String(transparent)}&toneMapped=${String(toneMapped)}`;
+  return `http://127.0.0.1:${window.remotion_proxyPort}/proxy?src=${encodeURIComponent(getAbsoluteSrc(src))}&time=${encodeURIComponent(Math.max(0, currentTime))}&transparent=${String(transparent)}&toneMapped=${String(toneMapped)}`;
 };
 
 // src/video/OffthreadVideoForRendering.tsx
diff --git a/dist/esm/no-react.mjs b/dist/esm/no-react.mjs
index 74d2850340ca03e2395eda5532dce30bd1e509b3..0e01dc31064a429781ed82dde221099c4e20a121 100644
--- a/dist/esm/no-react.mjs
+++ b/dist/esm/no-react.mjs
@@ -2516,7 +2516,7 @@ var getOffthreadVideoSource = ({
   currentTime,
   toneMapped
 }) => {
-  return `http://localhost:${window.remotion_proxyPort}/proxy?src=${encodeURIComponent(getAbsoluteSrc(src))}&time=${encodeURIComponent(Math.max(0, currentTime))}&transparent=${String(transparent)}&toneMapped=${String(toneMapped)}`;
+  return `http://127.0.0.1:${window.remotion_proxyPort}/proxy?src=${encodeURIComponent(getAbsoluteSrc(src))}&time=${encodeURIComponent(Math.max(0, currentTime))}&transparent=${String(transparent)}&toneMapped=${String(toneMapped)}`;
 };
 
 // src/no-react.ts
